### PR TITLE
ZCS-2592

### PR DESCRIPTION
**Pull request by mihaipopahtp**
- Use smart quotes (\u2019/ - \u2018/\u2019 ) instead of single quotes in translation files
- minor fixes with spacing/quote&formatting tag pairing
- fixed encoding for  src/zimlet/com_zimbra_about/com_zimbra_about.properties

### CODE REVIEW

**Review by silentsakky — CHANGES_REQUESTED** [review]
Please remove vietnamese files from here

**Comment by mihaipopahtp** [comment]
removed vi from all ZCS-2592 commits and squashed. 
Will update and squash on ZCS-1417

File: src/zimlet/com_zimbra_groupon/com_zimbra_groupon_iw.properties [hunk 1]
```diff
@@ -18,159 +18,4 @@ label = \u05d2\u05e8\u05d5\u05e4\u05d5\u05df
 preferences_title = \u05d4\u05e2\u05d3\u05e4\u05d5\u05ea
 preferences_saved = \u05d4\u05d4\u05e2\u05d3\u05e4\u05d5\u05ea \u05e0\u05e9\u05de\u05e8\u05d5
 showDailyDealsAsPopup = \u05d4\u05e6\u05d2 \u05de\u05d1\u05e6\u05e2\u05d9\u05dd \u05d9\u05d5\u05de\u05d9\u05d9\u05dd \u05d1\u05d7\u05dc\u05d5\u05df \u05e7\u05d5\u05e4\u05e5 (\u05e4\u05e2\u05dd \u05d1\u05d9\u05d5\u05dd)
-grouponError = '\u05d2\u05e8\u05d5\u05e4\u05d5\u05df' \u05d0\u05d9\u05e0\u05d5 \u05de\u05d2\u05d9\u05d1, \u05e0\u05e1\u05d4 \u05e9\u05d5\u05d1 \u05de\u05d0\u05d5\u05d7\u05e8 \u05d9\u05d5\u05ea\u05e8. HTTP
```
Comment (silentsakky) [inline src/zimlet/com_zimbra_groupon/com_zimbra_groupon_iw.properties:21]: lots of translations are removed from here, is it not needed?

**Review by silentsakky — APPROVED** [review]

File: src/zimlet/com_zimbra_groupon/com_zimbra_groupon_iw.properties (hunk 1 above)
Comment (mihaipopahtp) [reply src/zimlet/com_zimbra_groupon/com_zimbra_groupon_iw.properties:21]: that's an extra file that got stuck at the end of this one. I removed it and adding it to the next folder.

**Pull request merged**